helm/jiralert/templates/rbac.yaml

```
@@ -0,0 +1,33 @@
+---
+{{ if .Values.psp.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "resource.psp.name" . }}
```

```suggestion
  name: {{ include "resource.default.name" . }}
```
(or `-rbac`, or create an rbac resource name in _resource)